pkg/filemanager/manager/archive.go

```diff
@@ -37,6 +41,30 @@ func init() {
 	gob.Register([]ArchivedFile{})
 }
 
+func normalizeArchiveName(rawString string) string {
```

Comment:
This more looks like a "workaround" instead of a "solution", it will not work for non-Simplified-Chinese related encodings like `big5`. The not-so-good solution could be: Add a dropdown to let user select the encoding.